l10n_de_tax_statement/wizard/l10n_de_tax_statement_config_wizard.py

```diff
@@ -0,0 +1,232 @@
+        )
+        if config:
+            defv.setdefault('tag_41_base', config.tag_41_base.id)
+            defv.setdefault('tag_44_base', config.tag_44_base.id)
+            defv.setdefault('tag_49_base', config.tag_49_base.id)
+            defv.setdefault('tag_43_base', config.tag_43_base.id)
+            defv.setdefault('tag_48_base', config.tag_48_base.id)
+            defv.setdefault('tag_81_base', config.tag_81_base.id)
+            defv.setdefault('tag_86_base', config.tag_86_base.id)
+            defv.setdefault('tag_35_base', config.tag_35_base.id)
+            defv.setdefault('tag_36_tax', config.tag_36_tax.id)
+            defv.setdefault('tag_77_base', config.tag_77_base.id)
+            defv.setdefault('tag_76_base', config.tag_76_base.id)
+            defv.setdefault('tag_80_tax', config.tag_80_tax.id)
+            defv.setdefault('tag_91_base', config.tag_91_base.id)
+            defv.setdefault('tag_89_base', config.tag_89_base.id)
+            defv.setdefault('tag_93_base', config.tag_93_base.id)
+            defv.setdefault('tag_95_base', config.tag_95_base.id)
+            defv.setdefault('tag_98_tax', config.tag_98_tax.id)
+            defv.setdefault('tag_94_base', config.tag_94_base.id)
+            defv.setdefault('tag_96_tax', config.tag_96_tax.id)
+            defv.setdefault('tag_42_base', config.tag_42_base.id)
+            defv.setdefault('tag_68_base', config.tag_68_base.id)
+            defv.setdefault('tag_60_base', config.tag_60_base.id)
+            defv.setdefault('tag_21_base', config.tag_21_base.id)
+            defv.setdefault('tag_45_base', config.tag_45_base.id)
+            defv.setdefault('tag_46_base', config.tag_46_base.id)
+            defv.setdefault('tag_47_tax', config.tag_47_tax.id)
+            defv.setdefault('tag_52_base', config.tag_52_base.id)
+            defv.setdefault('tag_53_tax', config.tag_53_tax.id)
+            defv.setdefault('tag_73_base', config.tag_73_base.id)
+            defv.setdefault('tag_74_tax', config.tag_74_tax.id)
+            defv.setdefault('tag_78_base', config.tag_78_base.id)
+            defv.setdefault('tag_79_tax', config.tag_79_tax.id)
+            defv.setdefault('tag_84_base', config.tag_84_base.id)
+            defv.setdefault('tag_85_tax', config.tag_85_tax.id)
+            defv.setdefault('tag_65_tax', config.tag_65_tax.id)
+            defv.setdefault('tag_66_tax', config.tag_66_tax.id)
+            defv.setdefault('tag_61_tax', config.tag_61_tax.id)
+            defv.setdefault('tag_62_tax', config.tag_62_tax.id)
+            defv.setdefault('tag_67_tax', config.tag_67_tax.id)
+            defv.setdefault('tag_63_tax', config.tag_63_tax.id)
+            defv.setdefault('tag_64_tax', config.tag_64_tax.id)
+            defv.setdefault('tag_59_tax', config.tag_59_tax.id)
+            defv.setdefault('tag_69_tax', config.tag_69_tax.id)
+            return defv
+
+        if not self.is_l10n_de_coa_skr03() or not self.is_l10n_de_coa_skr04():
```

Comment:
```suggestion
        if not (self.is_l10n_de_coa_skr03() or self.is_l10n_de_coa_skr04()):
```

Could you check with this?

Reply:
@astirpe Now it is perfect! All cases are working (both german account charts + configurable account chart). Thanks a lot for your intervention!!